Documentation/specs/runtime-configuration-file.md

```diff
@@ -2,7 +2,7 @@
 
 The runtime configuration files store the dependencies of an application (formerly stored in the `.deps` file). They also include runtime configuration options, such as the Garbage Collector mode. Optionally they can also include data for runtime compilation (compilation settings used to compile the original application, and reference assemblies used by the application).
 
-**Note:** This document doesn't provide full explanations as to why individual items are needed in this file. That is covered in the [host spec](corehost.md) and via the `Microsoft.Extensions.DependencyModel` assembly.
+**Note:** This document doesn't provide full explanations as to why individual items are needed in this file. That is covered in the [host documentation](https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/host-component-dependencies-resolution.md) and via the `Microsoft.Extensions.DependencyModel` assembly.
```

Comment:
The `host-component-dependencies-resolution.md` is not general description of `.deps.json` handling... see my other comment for details.